README.md

```diff
@@ -504,6 +504,12 @@ Before following these instructions, ensure the existing model you are adding sk
    ilab data generate --endpoint-url http://localhost:8000/v1
    ```
 
+Note that it is also possible to generate a synthetic dataset based on the entire contents of the taxonomy repo using the `--taxonomy-base=empty` option:
+
+   ```shell
+   ilab data generate --taxonomy-base=`empty`
```

Comment:
nit: do we need the quotes around "empty"?

Reply:
this might break if users try to copy paste this command as is 
```
(venv) ➜  instructlab git:(ad84ac9) ilab taxonomy diff --taxonomy-base=`empty`
zsh: command not found: empty
compositional_skills/STEM/math/propositional_logic/arguments/qna.yaml
Taxonomy in /Users/jrao/Library/Application Support/instructlab/taxonomy is valid :)
```